xtask/src/main.rs

```diff
@@ -0,0 +1,57 @@
+use std::env;
```

Comment:
Put this stuff also in the main crate, say if we export a new command people will have to modify their crate. Make the main of this crate call the main of `fastn-xtask`.